samples/nrf9160/modem_shell/overlay-debug.conf

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
Something else we should add here? I'm not trying to capture everything but the common set people.

Reply:
how about CONFIG_NRF_CLOUD_GPS_LOG_LEVEL_DBG=y ?

Reply:
Yep, I was supposed to check nRF Cloud related different config flags but I forgot that. I'll add:
CONFIG_NRF_CLOUD_GPS_LOG_LEVEL_DBG=y
CONFIG_NRF_CLOUD_REST_LOG_LEVEL_DBG=y

Reply:
And generally, this file has been taken from ATv2 and irrelevant stuff removed and a few logs added.